plugin-tester-java/pom.xml

```diff
@@ -13,7 +13,9 @@
   <properties>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <akka.http.cors.version>0.4.2</akka.http.cors.version>
+    <maven-dependency-plugin.version>3.1.2</maven-dependency-plugin.version>
+    <maven-exec-plugin.version>3.0.0</maven-exec-plugin.version>
+    <akka.http.cors.version>1.1.0</akka.http.cors.version>
```

Comment:
In other places we're at 1.0.0, perhaps good to keep this consistent everywhere?

Reply:
indeed, I will do 1.1.0 everywhere.